ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -224,6 +224,15 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-network-mtu-edge-zone
+- as: aws-ipi-outpost-day2-f14
```

Comment:
I searched the email about new ci profiles in 4.15 and found followings. The process was introduced in late 4.15 timeframe so there might be some new profiles w/o email notification. I'd suggest to grep the new profiles by comparing 4.15 e2e test yaml to 4.14 e2e test yaml.

ibmcloud-ipi-disconnected-private, aws-ipi-wavelength-byo-subnet-ovn, aws-ipi-wavelength-fips, aws-ipi-wavelength-byo-subnet-ovn-day2, azure-ipi-fullyprivate-internal-registry, azure-ipi-mixed-apiserver-internal, azure-ipi-mixed-ingress-internal, azure-ipi-usertags-sa-encryption

Reply:
I will add them with another JIRA task https://issues.redhat.com/browse/OCPQE-20201